docs/README.md

```diff
@@ -19,5 +28,5 @@ See the following links for more on each of the resources involved:
 
 ## Getting Started Tasks
 
-- [Create an Ingress on the EventListener Service](create-ingress.yaml)
-- [Create a GitHub webhook](create-webhook.yaml)
+- [Create an Ingress on the EventListener Service](https://github.com/tektoncd/triggers/blob/master/docs/create-ingress.yaml)
```

Comment:
Having the branch hard coded here would break links to any new versioned docs that we have here: https://github.com/tektoncd/triggers#read-the-docs

Do we need to keep the absolute urls forever or can that be changed  back when its part of the website?

Reply:
Definitely short term.
Let me see if I can workaround that as everything under docs should come as a unit. The "sync" is currently very simple and does not pick-up files that are not markdown.

Reply:
One option is to leave it as a broken link in the website until we can figure out the right thing to do. I have no problem err-ing on the side of keeping our current github doc pristine.

Reply:
Cool, let's keep the relative links for now then.